src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -196,86 +190,47 @@
                                           :style {:width 500}
-                                          :onChange #(refresh-autocomplete {:engine engine
-                                                                            :workspace-attributes workspace-attributes
-                                                                            :entity-types entity-types
-                                                                            :selected-entity-type (.. % -target -value)})}
+                                          :onChange #(swap! state assoc :autocomplete-list
+                                                            (build-autocomplete-list
+                                                             {:workspace-attributes workspace-attributes
+                                                              :entity-types entity-types
+                                                              :selected-entity-type (.. % -target -value)}))}
                                          common/root-entity-types)
            [:div {:style {:padding "0.5em 0 1em 0"}} (:rootEntityType config)]))
-        (create-section-header "Inputs")
-        (this :-render-input-output-list
-              {:values (:inputs config)
-               :all-values (:inputs inputs-outputs)
-               :ref-prefix "in"
-               :invalid-values (:invalidInputs loaded-config)})
-        (create-section-header "Outputs")
-        (this :-render-input-output-list
-              {:values (:outputs config)
-               :all-values (:outputs inputs-outputs)
-               :ref-prefix "out"
-               :invalid-values (:invalidOutputs loaded-config)})]))
-   :-render-input-output-list
-   (fn [{:keys [state locals]}
-        {:keys [values ref-prefix invalid-values all-values]}]
-     (let [{:keys [editing?]} @state]
-       (create-section
-        (map
-         (fn [{:keys [name inputType outputType optional]}]
-           (let [type (or inputType outputType)
-                 name-kwd (keyword name)
-                 field-value (get values name-kwd "")
-                 error (get invalid-values name-kwd)]
-             [:div {:key name :style {:marginBottom "1rem"}}
-              (list
-               [:div {:style {:display "inline-block"
-                              :margin "0 0.5rem 0.5rem 0" :padding "0.5rem"
-                              :backgroundColor (:background-light style/colors)
-                              :border style/standard-line :borderRadius 2}}
-                (str name ": (" (when optional "optional ") type ")")]
-               (when (and error (not editing?) (not optional))
-                 (icons/icon {:style {:marginRight "0.5rem" :alignSelf "center"
-                                      :color (:exception-state style/colors)}}
-                             :error))
-               (when editing?
-                 [comps/Typeahead {:ref (str ref-prefix "_" name)
```

Comment:
The only other place we use `Typeahead` is in the library wizard. Wondering if we should add a story for migrating that to `TagAutocomplete` and removing  `Typeahead`.

Reply:
I would be in favor of that.

Reply:
+1 please write the story!

Reply:
Filed as GAWB-2544

Reply:
Created as GAWB-2553